projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -26,6 +26,8 @@ deploymentBuilderImage:
   registry: registry.hub.docker.com/versatiledatakit
   repository: job-builder
   tag: "1.2.3"
+  username:
```

Comment:
nit: being a bit more obvious in the same is not a bad thing. WHy not use registryUsernameReadOnly as above ?

Reply:
I actually think it is cleaner this way. 
They know it is builder registry because of the outer domain context. 
Let me know what you think?

Reply:
Ok.